packages/app/ui/components/Wayfinding/SplashSequence.tsx

```diff
@@ -340,10 +340,24 @@ export function InvitePane(props: { onActionPress: () => void }) {
     try {
       setIsProcessing(true);
       await store.syncSystemContacts();
-      // if successful, continue
-      props.onActionPress();
+      Alert.alert('Success', 'Your contacts have been synced.', [
+        {
+          text: 'OK',
+          onPress: () => {
+            props.onActionPress();
+          },
+        },
+      ]);
     } catch (error) {
       setError('Something went wrong, please try again.');
```

Comment:
Let's throw a PostHog log here.

Reply:
It's already covered in the try/catch here: https://github.com/tloncorp/tlon-apps/blob/fe3493dbe42d620638858a16bbb278b67ae2adec/packages/shared/src/store/sync.ts#L261